lib/telemetry_metrics_cloudwatch/cache.ex

```diff
@@ -229,4 +233,21 @@ defmodule TelemetryMetricsCloudwatch.Cache do
       "None"
     end
   end
+
+  defp get_storage_resolution(reporter_options) do
+    case Keyword.get(reporter_options, :storage_resolution, :standard) do
+      :high ->
+        1
+
+      :standard ->
+        60
+
+      other ->
+        Logger.error(
+          "Could not process storage_resolution #{inspect(other)}. Falling back to default."
+        )
```

Comment:
I think Cloudfront will likely return an error if we attempt to send an invalid value.  I think this is one place where a `raise` may be warranted.

Reply:
Done.